module/MotionParser.cpp

```diff
@@ -111,7 +111,50 @@ vector<Motion*> MotionParser::createMotions(const char* commandFilePath, int tar
       motionList.push_back(xr);                                          // 動作リストに追加
     }
     */
-    else {  // 未定義のコマンドの場合
+    else if(command == COMMAND::IS) {  // サークルの交点から交点までの直進
```

Comment:
```suggestion
    else if(command == COMMAND::IS) {  // 交点内移動（直進）
```
「交点サークル」と「サークルの交点」って紛らわしいからできれば別の言葉で...